package.json

```diff
@@ -1,19 +1,20 @@
 {
   "name": "busterbunny",
-  "version": "0.4.1",
-  "description" : "Opinionated EventBus Library for amqplib",
+  "version": "0.4.2",
```

Comment:
This should be a major version bump since the new validation may break existing callers that are only supplying an eventId and data.